torch/testing/_internal/distributed/distributed_test.py

```diff
@@ -6686,7 +6686,11 @@ class Bar:
 
             b = Bar()
             gather_objects = [b for _ in range(dist.get_world_size())]
-            with self.assertRaisesRegex(AttributeError, "Can't pickle local object"):
+            if sys.version_info >= (3, 12):
```

Comment:
@H-Huang tentatively tagging you for distributed tests. do we actually run distributed tests in CI with python 3.12?

Reply:
according to the HUD (https://hud.pytorch.org/) we only run them in 3.9 and 3.10